test/Concurrency/global_variables.swift

```diff
@@ -8,7 +8,7 @@
 
 @globalActor
 actor TestGlobalActor {
-  static var shared = TestGlobalActor()
```

Comment:
this code is now diagnosed as invalid since `shared` is global, mutable, non-isolated state. presumably this means this change could be a source-break – what is the stance on that when it comes to addressing bugs?

Reply:
It's okay to fix this as an error in Swift 6 mode - we can adjust if we find this causes wide spread source incompatibility amongst Swift 6 mode adopters through testing. These diagnostics are already warnings in strict concurrency checking / using the `GlobalConcurrency` upcoming feature prior to migrating to the Swift 6 language mode.